sources/packages/api/src/route-controllers/application/application.controller.ts

```diff
@@ -402,4 +384,30 @@ export class ApplicationController extends BaseController {
       formName: applicationProgramYear.programYear.formName,
     } as ProgramYearOfApplicationDto;
   }
+
+  /**
+   * API to fetch application details by applicationId and studentId.
+   * This API will be used by ministry users.
+   * @param applicationId
+   * @param userId
+   * @returns Application details
+   */
+  @AllowAuthorizedParty(AuthorizedParties.aest)
+  @Get("aest/:applicationId/:studentId")
```

Comment:
The URI here would be _api/application/aest/123/456/987654_, it does not look like the ones that we have been creating.
I believe that AEST/Student/Institution would be always at the most starting as possible.
I would assume that the API for this case would be something like
```
api/aest/student/123/application/456
```
@ann-aot @guru-aot thoughts?

Reply:
i agree it @andrewsignori-aot

Reply:
@dheepak-aot Yes, having an identifier before the actual values, will help us in better understanding of the APi, @andrewsignori-aot  I agree

Reply:
The API now looks like -> api/application/aest/123/student/456, api/application/aest/student/456 and api/student/aest/123.
To push aest before application and student, new controller is required. Given that discussion is going to happen on organizing the APIs, I want to keep the APIs in existing controllers itself. The pattern of continuous numbers in API is now fixed.